Dockerfile

```diff
@@ -22,40 +22,32 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-# Use Invenio's CentOS7 image with Python-3.6
-FROM docker.io/inveniosoftware/centos7-python:3.6
+# Use Invenio's alma image with Python-3.9
+FROM registry.cern.ch/inveniosoftware/almalinux:1
 
-# Use XRootD 4.12.7
-ENV XROOTD_VERSION=4.12.7
+# Use XRootD 5.6.3
+ENV XROOTD_VERSION=5.6.3
 
 # Install CERN Open Data Portal web node pre-requisites
 # hadolint ignore=DL3033
 RUN yum install -y \
         ca-certificates \
         cmake3 \
-        curl \
-        git \
+        emacs-nox \
```

Comment:
Thanks, removed. What about screen and rlwrap?
I would prefer to keep vim.

Reply:
We can keep `vim` for sure, just to be on the safe side. However, it was rarely a need to edit anything inside the container when it is running in production. And, if we are running locally for development, then we can always use volume mounting and use comfortably our host editor for that.

The `screen` ditto, it may be helpful for running longer-term tasks, however it was rarely a need in practice inside a running container.

The `rlwrap` could be useful though, for example for `ipython` shell history readline-like completion. I haven't checked whether the new base container already behaves well WRT ipython history completion; perhaps rlwrap is not necessary there. anymore.

Reply:
Thanks. Screeen removed then.